### ISSUE TITLE
"D" keyboard shortcut should not delete a cell

### ISSUE THREAD

**Issue by dnigendac** [rationalmatter/Juno-Issue-Tracker#47]
**Is your feature request related to a problem? Please describe.**
I realize this is also the case on the "old" Juno app, but it messes up with my muscle memory to have "D" delete the cell. I always end up doing "DD" like in normal Jupyter, and so I end up with two deleted cells.

**Describe the solution you'd like**
I'd like to remove the "D" shortcut and replace with "DD". Typing "DD" when in command mode should delete one cell.

**Comment by navoshta** [comment]
Thanks for reporting — it should be "DD", I will look into this.

**Comment by navoshta** [comment]
@dnigendac Juno 1.2 is now waiting for TestFlight review, and should be available in the next few days — please send an e-mail to beta@juno.sh if you would like to help test it! Delete cell shortcut is now "Shift + D" instead of "D" or "DD" — you can see all key bindings while pressing Command key.

**Comment by dnigendac** [comment]
Works! Shame it can't be "DD" like in normal Jupyter, but now it's harder to fat-finger delete a cell, which is great.

**Comment by navoshta** [comment]
@dnigendac Yep I'm still hoping to get back to it and add a proper "DD" shortcut, but this will involve a bit more work I'm afraid (as far as I know, iOS doesn't natively handle key bindings with multiple keys, so I will need to implement manual handling of this). I thought I'd just make it a bit harder to hit erroneously for now.
